style_traits/Cargo.toml

```diff
@@ -1,17 +1,20 @@
 [package]
-name = "style_traits"
-version = "0.0.1"
+name = "stylo_traits"
```

Comment:
I think we shouldn't rename crates downstream and instead this should be managed in coordination with upstream.